tests/integration/test_runtime_configurable_cache_size/test.py

```diff
@@ -101,9 +101,11 @@ def test_query_cache_size_is_runtime_configurable(start_cluster):
     node.query("SELECT 2 SETTINGS use_query_cache = 1, query_cache_ttl = 1")
     node.query("SELECT 3 SETTINGS use_query_cache = 1, query_cache_ttl = 1")
 
+    time.sleep(2)
     res = node.query_with_retry(
-        "SELECT value FROM system.asynchronous_metrics WHERE metric = 'QueryCacheEntries'",
+        "SELECT value FROM system.asynchronous_metrics WHERE metric = 'QueryCacheEntries' SETTINGS use_query_cache=0",
```

Comment:
`use_query_cache = 0` is the default anyways, this has no effect, please remove.